static/js/admin/scannerqueryresult.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener('DOMContentLoaded', () => {
+  if (!document.body.matches('.change-list.model-scannerqueryresult')) {
+    // This is only for the scanners query result change list page.
+    return;
+  }
+
+  let current_addon = {
+    cell: null,
+    link: null,
+  };
+
+  for (row of result_list.tBodies[0].rows) {
+    let new_addon = {
+      cell: row.querySelector('.field-formatted_addon'),
+      link: row.querySelector('.field-formatted_addon a'),
+    };
+
+    // FIXME: for performance, it might be better to store the elements we
+    // want to change in a list and change them in one go outside of this
+    // loop ?
```

Comment:
I suppose it's fine unless you have more than 1K elements probably?

Reply:
Yeah - and we paginate for 100, so it's likely fine (appears ok on my machine), but I left the comment for now just in case we want to revisit the pagination someday.